src/main/java/base/CopyVerifier.java

```diff
@@ -5,63 +5,73 @@
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.chrome.ChromeDriver;
 import org.testng.Assert;
+import utils.Utils;
 
 import java.util.concurrent.TimeUnit;
 
 public class CopyVerifier {
-
-    private static SheetsQuickstart sheets;
     private static WebDriver driver;
 
-
-    public CopyVerifier(){
+    public CopyVerifier() {
         driver = new ChromeDriver();
         driver.manage().timeouts().implicitlyWait(1, TimeUnit.SECONDS);
         driver.get("https://assistant.google.com/");
         //home = new Home_Page(driver);
     }
 
-    public String getCopy(String _selector){
+    public String getCopy(String _selector) {
         String dataType = "";
-        String [] selectorKeys = _selector.split(",");
+        String[] selectorKeys = _selector.split(",");
         String selector = selectorKeys[0];
-        if(selectorKeys.length>1) {
+        if (selectorKeys.length > 1) {
             dataType = selectorKeys[1];
         }
         WebElement element = driver.findElement(By.cssSelector(selector));
         String tag = element.getTagName();
-        System.out.println("tag " +tag);
+        System.out.println("tag " + tag);
 
-        switch (tag){
+        switch (tag) {
             case "img":
-                System.out.println("Case IMG " + element.getAttribute("data-src"));
-                return element.getAttribute("data-src");
-            case "h2":
-                System.out.println("Case H2 " + element.getAttribute("innerHTML").trim());
-                return element.getAttribute("innerHTML").trim();
+                System.out.println("Case IMG " + getCopyFromType(dataType, element));
+                return getCopyFromType(dataType, element);
             case "a":
-                System.out.println("Case a " + getCopyFromType(dataType,element));
-                return getCopyFromType(dataType,element);
-
+                System.out.println("Case a " + getCopyFromType(dataType, element));
+                return getCopyFromType(dataType, element);
+            case "div":
+                System.out.println("Case div " + element.getAttribute("data-c-video-id"));
+                String youTubeVideo = "https://youtube.com/watch?v=" + element.getAttribute("data-c-video-id");
+                return youTubeVideo;
+            case "span":
+            case "h3":
+            case "h2":
+            case "p":
+                System.out.println("Case " + tag + ": " + Utils.htmlToText(element.getAttribute("innerHTML").trim()));
+                return Utils.htmlToText(element.getAttribute("innerHTML").trim());
```

Comment:
here I removed `replaceAll("&nbsp;", " ")` because the HTML util text will take care of it, I tested it and it seems to be working as expected, but make sure you take a look to it too.